CMakeLists.txt

```
@@ -53,17 +53,17 @@ set(CTR_DRBG_128_BIT_KEY_WARNING "${WARNING_BORDER}"
 
 find_package(PythonInterp)
 find_package(Perl)
```

I assume this needs to be 
```suggestion
find_package(Python)
```

---

Well-spotted. `Perl` is no longer needed. But [`PythonInterp`](https://cmake.org/cmake/help/v3.5/module/FindPythonInterp.html) is already present. There's also a [`FindPython`](https://cmake.org/cmake/help/v3.12/module/FindPython.html) but only on recent CMake versions. And `PYTHON_FOUND` below should actually be `PYTHONINTERP_FOUND`.

---

And on my machine `FindPython` actually finds `python` which is Python 2 but we need Python 3!